deploy/sample-backup-configs/sample_config_build.ini

```diff
@@ -18,7 +18,8 @@ firesim-boom-singlecore-no-nic-l2-llc4mb-ddr3
 firesim-boom-singlecore-nic-l2-llc4mb-ddr3
 
 firesim-supernode-rocket-singlecore-nic-l2-lbp
-firesim-boom-singlecore-no-nic-l2-llc4mb-ddr3-ramopts
+#firesim-boom-singlecore-no-nic-l2-llc4mb-ddr3-ramopts
+firesim-rocket-quadcore-no-nic-l2-llc4mb-ddr3-halfrate
```

Comment:
I think we want a more descriptive name than halfrate

Reply:
Yeah if anything i kept it around because it makes me chuckle. 

half-frequency-uncore, 2xfreq-tiles Suggestions?

Reply:
half-freq-uncore
I think you want the nominal frequency to be the tile frequency, so 2xfreq-tiles is confusing.

Reply:
half-freq-uncore SGTM